docs/develop/enable-nested-app-authentication-in-your-add-in.md

```diff
@@ -61,7 +61,7 @@ The following steps show how to enable NAA in the `taskpane.js` or `taskpane.ts`
 
     ```json
     "dependencies": {
-        "@azure/msal-browser": "^3.15.0",
+        "@azure/msal-browser": "^3.24.0",
         ...
```

Comment:
Any idea why this is showing up weird in GitHub?

![image](https://github.com/user-attachments/assets/187d15df-b1f0-44db-a5bd-7e30dbe5c800)

Reply:
no it's odd.